consensus/state.go

```diff
@@ -1560,6 +1542,12 @@ func (cs *State) enterPrecommit(height int64, round int32) {
 		return
 	}
 
+	if ready, waitTime := cs.isReadyToPrecommit(); !ready {
+		logger.Info("rescheduling precommit", "delay(s)", waitTime.Seconds())
+		cs.scheduleTimeout(waitTime, height, round, cstypes.RoundStepPrevoteWait)
```

Comment:
This would trigger the `enterPrecommit`?

Reply:
yes